build/xtask/src/dist.rs

```diff
@@ -1497,15 +1560,7 @@ pub fn make_kconfig(
         .flat_map(|(_name, task)| task.uses.iter())
         .collect::<HashSet<_>>();
 
-    // ARMv6-M and ARMv7-M require that memory regions be a power of two.
-    // ARMv8-M does not.
-    let power_of_two_required = match toml.target.as_str() {
-        "thumbv8m.main-none-eabihf" => false,
-        "thumbv7em-none-eabihf" => true,
-        "thumbv6m-none-eabi" => true,
-        t => panic!("Unknown mpu requirements for target '{}'", t),
-    };
-
+    let power_of_two_required = toml.mpu_power_of_two_required();
```

Comment:
Given that this routine is already consulting the MPU requirement data, could you move the computation of `align` into here, rather than consulting the MPU requirement data outside this function and then calling it?